src/summary/history/history-section.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { match } from "react-router";
+import SummarySectionHeader from "../summary-section-header.component";
+import SummarySectionCards from "../cards/summary-section-cards.component";
+import AllergyCard from "./allergy-card.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+
+export default function HistorySection(props: HistorySectionProps) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient =>
+      setCurrentPatient(patient)
+    );
+
+    return () => subscription.unsubscribe();
+  });
+
+  return (
+    <>
+      <SummarySectionHeader match={props.match} name="History" />
```

Comment:
As of Greg's new 1.3.0 designs, there is no more SummarySectionHeader. Please remove this. See [designs in figma](https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart---Master-(current-file)?node-id=1550%3A30065) for more info.